auth/httptransport/httptransport.go

```diff
@@ -170,10 +171,14 @@ func AddAuthorizationMiddleware(client *http.Client, creds *auth.Credentials) er
 			base = http.DefaultTransport
 		}
 	}
+	clientUniverseDomain, err := creds.UniverseDomain(context.Background())
```

Comment:
This now potentially can hard pull MDS, or at least could in the future. Can this be delayed?

Reply:
Fixed, thanks for catching this.